wordpress-mobile/gutenberg-mobile
# Ensure that empty posts are not published

Fixes #369

This just updates the GB reference in order that empty posts autosaves are sent to the bridge method.

Check this GB PR for the changes: https://github.com/WordPress/gutenberg/pull/23233

To test:
 - Test using this WP-iOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14329
 - Create a new post
 - Write some text on a paragraph
 - Remove the paragraph using the block toolbar trash option
 - Make sure the publish button is disabled when the post is empty

PR submission checklist:
